README.md

```diff
@@ -59,6 +59,70 @@ f1.content = "The quick brown fox jumped over the lazy dog."
 bf1.save
 ```
 
+## Extending Directly Contained Derivatives
+
+Hydra Works comes with the ability to attach [Hydra Derivatives](https://github.com/projecthydra/hydra-derivatives) to a GenericFile. This allows a GenericFile of an image, video, audio file, or document type to directly contain appropriate derivative files such as thumbnail, extracted_text, mp3, ogg, mp4, and webm. The derivatives are configured in `lib/hydra/works/models/concerns/generic_file/contained_files.rb` and `lib/hydra/works/models/concerns/generic_file/derivatives.rb`.
+
+More directly contained derivatives can be configured for the Persist Derivatives service by extending `contained_files` and `makes_derivatives`, as long as the processor is defined in Hydra Derivatives.
```

Comment:
I'd also strike the reference to the `Persist Derivatives` service, and say something like this here:

```More derivatives can be generated by extending `contained_files` and `makes_derivatives`, as long as  processors for them are defined in Hydra::Derivatives (linked above).```